discovery/pkg/openstack/httplogger.go

```diff
@@ -57,7 +57,7 @@ func (lrt *LogRoundTripper) RoundTrip(request *http.Request) (*http.Response, er
 
 	if response.StatusCode == http.StatusUnauthorized {
 		if lrt.numReauthAttempts == 3 {
-			return response, fmt.Errorf("Tried to re-authenticate 3 times with no success.")
+			return response, fmt.Errorf("Tried to re-authenticate 3 times with no success")
```

Comment:
While you're here could you please s/Tried/tried, the rationale is often times errors are concatenated together; ie `could not create openstack provider: Tried to re-authenticate ...` << the Capitalisation in the middle of the sentence looks odd.

Maybe just rewrite the whole thing to

```
fmt.Errorf("tried to re-authenticate 3 times without success")
```

Reply:
Thank you for reviewing.
I see, I'll fix the point you pointed out